pages/common/content/index.js

```diff
@@ -29,7 +29,8 @@ module.exports = {
           confirm: 'Confirm',
           success: 'Success'
         }
-      }
+      },
+      permission: 'Change / remove permissions'
```

Comment:
this feels like it should be under profile

Reply:
It's a breadcrumb label - can't see any other breadcrumb content outside of this file?